app/overrides/add_reviews_after_product_properties.rb

```diff
@@ -1,6 +0,0 @@
-Deface::Override.new(
-  virtual_path: 'spree/products/show',
-  name: 'converted_product_properties_767643482',
-  insert_after: '[data-hook="product_properties"]',
-  partial: 'spree/shared/reviews'
-)
```

Comment:
<img width="1171" alt="image" src="https://github.com/spreehood/spree_reviews/assets/6129641/92325649-c74a-4100-860c-94a26bf93db9">
we should have reviews after product properties, can you make this tab menu override?
@mausamp